lib/webpack.config.js

```diff
@@ -46,6 +46,26 @@ const babelLoader = (plugins = []) => ({
   },
 })
 
+const packageWhitelist = ['catalog-ui-search', 'imperio', '@connexta']
```

Comment:
I think we should move towards grabbing the whitelist from the `package.json` of whoever is using ace.